integration/compatibility/generate.py

```
@@ -44,7 +45,7 @@ def setup_class(cls):
         if cls.ANSWER_FILE_NAME is None:
             raise NotImplementedError("Subclass must define ANSWER_FILE_NAME")
             
-        if os.system("docker run --rm -d --name Generate-search -p 6380:6379 redis/redis-stack-server") != 0:
+        if os.system("docker run --rm -d --name Generate-search -p 6380:6379 redis") != 0:
```

Ruff (0.15.14)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)